app/src/main/res/values/strings.xml

```diff
@@ -206,6 +206,7 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="unordered_list_char_asterisk">Asterisk (*)</string>
     <string name="unordered_list_char_plus">Plus (+)</string>
     <string name="filemanager_doesnot_supply_required_data__appspecific">Markor can only edit local, offline available documents and does not support file managers that do not provide the full file path.</string>
+    <string name="file_does_not_exist">File or directory does not exist</string>
```

Comment:
Warn that intent files don't exist (most common case is shortcut which has been deleted)